static/js/github-create-branch.js

```diff
@@ -90,34 +90,24 @@ const loadBranches = () => {
     type: "GET",
     url: `/github/create-branch/owners/${repo.owner}/repos/${repo.name}/branches`,
     success: (data) => {
-      const allBranchesfetched = data?.repository?.refs?.totalCount === data?.repository?.refs?.edges.length;
       $("#ghParentBranch").auiSelect2({
         data: () => {
-          data.repository.refs.edges.forEach((item) => {
-            item.id = item.node.name;
+          data.forEach((item) => {
+            item.id = item.name;
           });
           return {
-            text: item => item.node.name,
-            results: data.repository.refs.edges
+            text: item => item.name,
+            results: data
           }
         },
-        formatSelection: item => item.node.name,
-        formatResult: item => item.node.name,
-        createSearchChoice: (term) => {
-          if (allBranchesfetched) {
```

Comment:
why you remove `createSearchChoice`?

Reply:
This was removed because we do not have `allBranchFetched`. The Graphql doesn't return the total number of repos.

Reply:
Earlier it is allowing to create new choice only in case if all branches are not fetched.  If we're unable to identify total number of repos, then it make sense to allow it in every case.

Reply:
Reverted this back excluding the `if` case.